packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -1,441 +1,19 @@
-    unlockedBalance: 0,
-    lockedBalance: 2500,
-    unlockedAuthorities: 0,
-    lockedAuthorities: 0,
-    timelockExpires: timelock,
-    transactions: 1,
-  }]);
-
-  // let's now receive a tx that spends this utxo, while it's still marked as locked
-  const evt = JSON.parse(JSON.stringify(eventTemplate));
-  const tx = evt.Records[0].body;
-  tx.version = 1;
-  tx.tx_id = txId2;
-  tx.timestamp += timelock + 1;
-  tx.inputs = [createInput(2500, addr, txId1, 0, token)];
-  tx.outputs = [
-    createOutput(0, 2000, addr, token),    // one output to the same address
-    createOutput(1, 500, 'other', token),  // and one to another address
-  ];
-  await txProcessor.onNewTxEvent(evt);
-
-  await expect(checkUtxoTable(mysql, 2, txId2, 0, token, addr, 2000, 0, null, null, false)).resolves.toBe(true);
-  await expect(checkUtxoTable(mysql, 2, txId2, 1, token, 'other', 500, 0, null, null, false)).resolves.toBe(true);
-  await expect(checkAddressTable(mysql, maxGap + 2, addr, 0, walletId, 2)).resolves.toBe(true);
-  await expect(checkAddressTable(mysql, maxGap + 2, 'other', null, null, 1)).resolves.toBe(true);
-  await expect(checkAddressBalanceTable(mysql, 2, addr, token, 2000, 0, null, 2)).resolves.toBe(true);
-  await expect(checkAddressBalanceTable(mysql, 2, 'other', token, 500, 0, null, 1)).resolves.toBe(true);
-  await expect(checkWalletBalanceTable(mysql, 1, walletId, token, 2000, 0, null, 2)).resolves.toBe(true);
-});
-
-test.skip('Genesis transactions should throw', async () => {
-  expect.hasAssertions();
-
-  const evt = JSON.parse(JSON.stringify(eventTemplate));
-  const tx = evt.Records[0].body;
-
-  tx.inputs = [];
-  tx.outputs = [];
-  tx.parents = [];
-
-  process.env.NETWORK = 'mainnet';
-
-  tx.tx_id = txProcessor.IGNORE_TXS.mainnet[0];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-
-  tx.tx_id = txProcessor.IGNORE_TXS.mainnet[1];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-
-  tx.tx_id = txProcessor.IGNORE_TXS.mainnet[2];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-
-  process.env.NETWORK = 'testnet';
-
-  tx.tx_id = txProcessor.IGNORE_TXS.testnet[0];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-
-  tx.tx_id = txProcessor.IGNORE_TXS.testnet[1];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-
-  tx.tx_id = txProcessor.IGNORE_TXS.testnet[2];
-
-  await expect(() => txProcessor.onNewTxEvent(evt)).rejects.toThrow('Rejecting tx as it is part of the genesis transactions.');
-});
-
-/*
- * receive some transactions and blocks and make sure database is correct
- */
-test.skip('txProcessor', async () => {
```

Comment:
We have integration tests that run the simulation in the fullnode and check if balances are correct after a full sync which is way better than mocking it. So no need to re-implement this test.